dockers/docker-dhcp-server/cli/config/plugins/dhcp_server.py

```diff
@@ -252,12 +252,93 @@ def dhcp_sever_ipv4_range_del(db, range_name, force):
             for port in dbconn.keys("CONFIG_DB", "DHCP_SERVER_IPV4_PORT*"):
                 ranges = dbconn.get("CONFIG_DB", port, "ranges")
                 if ranges and range_name in ranges.split(","):
-                    ctx.fail("Range {} is referenced in {}, cannot delete, add --force to bypass".format(range_name, port))
+                    ctx.fail("Range {} is referenced in {}, cannot delete, add --force to bypass or range unbind to unbind range first".format(range_name, port))
         dbconn.delete("CONFIG_DB", key)
     else:
         ctx.fail("Range {} does not exist, cannot delete".format(range_name))
 
 
+@dhcp_server_ipv4.command(name="bind")
+@click.argument("dhcp_interface", required=True)
+@click.argument("member_interface", required=True)
```

Comment:
Suggest adding checks to make sure member interface is in dhcp_interface. (In current scenario we can check `VLAN_MEMBER` table)